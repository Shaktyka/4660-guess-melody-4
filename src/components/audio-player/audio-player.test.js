import React from 'react';
import renderer from 'react-test-renderer';
import AudioPlayer from './audio-player.jsx';

const SRC = `https://www.youtube.com/audiolibrary_download?vid=79100e44c826e2f7`;
const IS_PLAYING = false;

describe(`AudioPlayer rendering`, () => {

  it(`AudioPlayer renders correctly`, () => {
    const tree = renderer.create(
        <AudioPlayer
          isPlaying={IS_PLAYING}
          onPlayButtonClick={() => {}}
          src={SRC}
        />, {
          createNodeMock: () => {
            return {};
          }
        }
    ).toJSON;

    expect(tree).toMatchSnapshot();
  });

});
