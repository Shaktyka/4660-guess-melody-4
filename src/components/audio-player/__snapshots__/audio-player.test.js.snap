// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPlayer rendering AudioPlayer renders correctly 1`] = `[Function]`;
